.github/licenserc.yml

```diff
@@ -8,7 +8,7 @@ header:
     - '.gitignore'
     - '.gitattributes'
     - '.golangci.yml'
-    - '.licenserc.yaml'
+    - 'licenserc.yml'
```

Comment:
```suggestion
```

This can can be removed, because the config file have moved to the `.github/`

Reply:
Updated.